diff --git a/node_modules/vite-plugin-markdown-preview/dist/index.js b/node_modules/vite-plugin-markdown-preview/dist/index.js
index 5c354f7..65b9437 100644
--- a/node_modules/vite-plugin-markdown-preview/dist/index.js
+++ b/node_modules/vite-plugin-markdown-preview/dist/index.js
@@ -13,6 +13,7 @@ import { createHash } from "node:crypto";
 import { fromMarkdown } from "mdast-util-from-markdown";
 import { frontmatterFromMarkdown, frontmatterToMarkdown } from "mdast-util-frontmatter";
 import { toMarkdown } from "mdast-util-to-markdown";
+import { frontmatter } from 'micromark-extension-frontmatter';
 import { visit } from "unist-util-visit";
 function getHash(text) {
   return createHash("sha256").update(text).digest("hex").substring(0, 8);
@@ -28,6 +29,7 @@ function praseMeta(meta) {
 }
 function remarkDemoBlock(id, code, config) {
   const tree = fromMarkdown(code, {
+    extensions: [frontmatter(['yaml', 'toml'])],
     mdastExtensions: [frontmatterFromMarkdown(["yaml", "toml"])]
   });
   const blocks = {};
